Extend/Controller/OrderController.php

```diff
@@ -369,13 +371,52 @@ private function _getCreditCardFormRequestData()
 
         $oPayment = PaymentMethodHelper::getPaymentById($oBasket->getPaymentId());
         $sPaymentAction = $this->_getPaymentAction($oPayment->oxpayments__wdoxidee_transactionaction->value);
-        $sLanguageCode = Registry::getLang()->getLanguageAbbr();
+        $sLanguageCode = $this->_getSupportedLanguageCode();
 
         $oTransactionService = $this->_getTransactionService();
 
         return $oTransactionService->getCreditCardUiWithData($oTransaction, $sPaymentAction, $sLanguageCode);
     }
 
+    /**
+     * Gets the supported WPP language code
+     *
+     * @return string
+     *
+     * @since 1.3.0
+     */
+    private function _getSupportedLanguageCode()
+    {
+        $wConverter = new WppVTwoConverter();
+        try {
+            $wConverter->init();
+            $wLanguage = $wConverter->convert(
+                $this->_removeSuffix(Registry::getLang()->getLanguageAbbr())
```

Comment:
@sebastijansakac maybe this is too complex and would be better to do smth like:
$langAbbr = $this->_removeSuffix(Registry::getLang()->getLanguageAbbr());
and then
$wLanguage = $wConverter->convert($langAbbr);
but its just a suggestion.

Reply:
Updated as suggested, and to keep consistence with PS